cmd/clusterctl/examples/azure/machines.yaml.template

```diff
@@ -10,8 +10,8 @@ items:
         set: controlplane
     spec:
       versions:
-        kubelet: 1.12.5
-        controlPlane: 1.12.5
+        kubelet: 1.13.3
```

Comment:
Super nit and unimportant as far as this PR is concerned: Should we look into bringing the client-go dependencies into parity with the k8s version?

Reply:
Let's tackle adding a constraint for that in a follow-on PR: https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/93